frontend/devServer.js

```diff
@@ -4,6 +4,10 @@ const ParcelProxyServer = require("parcel-proxy-server")
 // https://github.com/charlieduong94/parcel-proxy-server
 const server = new ParcelProxyServer({
   entryPoint: "./public/index.html",
+  // makes rebuild in dev less taxing V
+  // parcelOptions: {
+  //   sourceMaps: false,
+  // },
```

Comment:
Is this an option we can keep around or should we nix it completely

Reply:
Im still trying to get my head around the parcel module builder. when i put that in the hot module reload was making my computer go crazy, and this was a suggested fix. I didn't want to leave it live bc i don't know the side-effects of it. I can remove.